### PR TITLE
Disk errors handling

### PR DESCRIPTION
**Pull request by PMazarovich**
Once after IO error worker was killed and all events just began to accumulate endlessly.
That is why it is better to handle disk errors to keep worker alive.

### CODE REVIEW

File: logstash_async/database.py [hunk 1]
```diff
@@ -29,6 +29,10 @@ class DatabaseLockedError(Exception):
     pass
 
 
+class DiskIOError(Exception):
```
Comment (eht16) [inline logstash_async/database.py:32]: Could we name it `DatabaseDiskIOError` so it is inline with `DatabaseLockedError`?

File: logstash_async/database.py [hunk 2]
```diff
@@ -96,6 +100,13 @@ def _handle_sqlite_error(self):
         _, exc, _ = sys.exc_info()
         if str(exc) == 'database is locked':
             raise DatabaseLockedError from exc
+        if str(exc) == 'disk I/O error':
```
Comment (eht16) [inline logstash_async/database.py:103]: Did you take these message texts from real errors or is there some reference?

File: tests/database_test.py [hunk 3]
```diff
@@ -63,6 +64,13 @@ def close_connection(cls):
             cls._connection.close()
         cls._connection = None
 
+    def testIOException(self):
```
Comment (eht16) [inline tests/database_test.py:67]: Great!

**Review by eht16 — CHANGES_REQUESTED** [review]
Great, thank you!

I have just a couple of minor comments.
And could you have a look at the code style, basically PEP8 + 100 charcters line length (see the failed CI for details) or test locally by running `tox`.

Thanks!

**Comment by PMazarovich** [comment]
@eht16 requested changes are in place

File: logstash_async/database.py (hunk 2 above)
Comment (PMazarovich) [reply logstash_async/database.py:102]: @eht16 yes, these were copied from real exception messages

**Pull request merged**